tests/tensor/rewriting/test_shape.py

```diff
@@ -419,6 +426,60 @@ def test_basic(self):
 
         assert check_stack_trace(g, ops_to_check=(DimShuffle, Reshape))
 
+    def test_expand_dims(self):
+        x = pt.scalar()
+        # This reshape does an implicit expand_dims
+        out = x.reshape((1, -1))
+        assert isinstance(out.owner.op, Reshape)
+        new_out = rewrite_graph(out, include=("canonicalize",))
+        assert equal_computations([new_out], [pt.expand_dims(x, (0, 1))])
```

Comment:
Should you check the presence/absence of specific ops after the rewrite? Or does `equal_computations` handle that?

Reply:
It does, they must match exactly to pass